relay-profiling/src/sample/v1.rs

```diff
@@ -354,6 +358,16 @@ pub fn parse_sample_profile(
         }
     }
 
+    profile.metadata.client_sdk = match (
+        transaction_metadata.get("client_sdk.name"),
+        transaction_metadata.get("client_sdk.version"),
+    ) {
+        (Some(name), Some(version)) => Some(ClientSdk {
+            name: name.to_owned(),
+            version: version.to_owned(),
+        }),
+        _ => None,
+    };
```

Comment:
Should we add this also for `v2` where the field already exists but doesn't seem to be populated?

Reply:
No, this isn't possible for `v2` because the `v2` payload will be sent in a separate envelope from the transaction making it impossible to populate this field from the transaction.